### PR TITLE
Clarify the meaning of the fields for Gift Card Information block

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) clarifying  the meaning of the fields for Gift Card Information block

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/product-gift-card-information.html

### CODE REVIEW

File: src/catalog/product-gift-card-information.md [hunk 1]
```diff
@@ -7,3 +7,10 @@ The _Gift Card Information_ section can be used to override the [configuration](
 
 ![]({% link images/images-ee/product-gift-card-information.png %}){: .zoom}
 [_Gift Card Information_]({% link catalog/product-gift-card-create.md %})
+
+|Field|Description|
+|--- |--- |
+| Treating Balance As Store Credit | Determines if the gift card holder can repay the balance as a store credit. |
+| Validity period (days) | Defines the expiration date of the gift card in days. You can leave the field empty if you do not want to set a limit on the card's validity period, leave this field empty. |
+| Allow message | Determines the ability to enter a message for the recipient of the gift card. A gift message can be enabled for both virtual (sent by email) and physical (sent) gift cards. |
+| Email Template | Defines the email template that is used for the notification sent to the recipient of the gift card. |
```
Comment (leslietilling) [inline src/catalog/product-gift-card-information.md:16]:
```suggestion
| Treating Balance As Store Credit | Determines if the gift card holder can redeem the balance as store credit. |
| Validity period (days) | Defines the expiration date of the gift card as number of days after purchase. You can leave the field empty if you do not want the card to expire. |
| Allow message | Determines the ability of the purchaser to enter a message for the recipient of the gift card. A gift message can be enabled for both virtual (sent by email) and physical (sent) gift cards. |
| Email Template | Defines the email template that is used for the notification sent to the recipient of the gift card. |

```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits for style and clarity

**Comment by EvheniiaKolkhiieva** [comment]
Thank you for reviewing @leslietilling , done, please approve ☀️

File: src/catalog/product-gift-card-information.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-gift-card-information.md]: done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**